mars/utils.py

```diff
@@ -1629,6 +1634,18 @@ def cache_tileables(*tileables):
             t.cache = True
 
 
+def report_event(severity, label, message):
```

Comment:
If it is only for Ray, put it into "mars/oscar/backends/ray/utils.py".

Reply:
The ray executor can use it to report event too.

Reply:
Can import from "mars/oscar/backends/ray/utils.py".

Reply:
moved to `mars/oscar/backends/ray/utils.py`